tests/unit/test_bet.py

```diff
@@ -163,6 +163,63 @@ def test_repr_names(bet, bet_name):
     assert repr(bet) == bet_name
 
 
+# fmt: off
+def test_str_names():
+    bets = [
+        (crapssim.bet.PassLine(5), "$5 PassLine"),
+        (crapssim.bet.Come(1), "$1 Come"),
+        (crapssim.bet.Come(1, 6), "$1 Come(6)"),
+        (crapssim.bet.DontPass(1), "$1 DontPass"),
+        (crapssim.bet.DontCome(1), "$1 DontCome"),
+        (crapssim.bet.DontCome(1, 4), "$1 DontCome(4)"),
+        (crapssim.bet.Odds(crapssim.bet.PassLine, 6, 1, False), "$1 Odds(PassLine)"),
+        (crapssim.bet.Odds(crapssim.bet.Come, 8, 1), "$1 Odds(Come, 8)"),
+        (crapssim.bet.Odds(crapssim.bet.DontPass, 9, 1), "$1 Odds(DontPass)"),
+        (crapssim.bet.Odds(crapssim.bet.DontCome, 10, 1), "$1 Odds(DontCome, 10)"),
+        (crapssim.bet.Odds(crapssim.bet.PassLine, 6, 1, True), "$1 Odds(PassLine)"),
+        (crapssim.bet.Odds(crapssim.bet.Come, 8, 1, True), "$1 Odds(Come, 8)"),
+        (crapssim.bet.Odds(crapssim.bet.DontPass, 9, 1, True), "$1 Odds(DontPass)"),
+        (crapssim.bet.Odds(crapssim.bet.DontCome, 10, 1, True), "$1 Odds(DontCome, 10)",),
```

Comment:
Note that the `always_working` argument is currently not captured in the string representation. 

I think this is okay in general, as `__str__` does not need to include all arguments. If in doubt, or for some debugging within a strategy, use `__repr__` like `repr(bet)` or `f"Bet: {bet!r}"`.